invenio_records_marc21/cli.py

```diff
@@ -41,6 +48,28 @@ def _load_json(filename):
         return json.load(fp)
 
 
+def create_fake_metadata():
+    """Create records for demo purposes."""
+    metadata = Metadata()
+    metadata.xml = "<record><controlfield tag='001'>990079940640203331</controlfield><controlfield tag='007'>cr</controlfield> <controlfield tag='008'>100504|1932</controlfield><controlfield tag='009'>AC08088803</controlfield><datafield tag='035' ind1=' ' ind2=' '><subfield code='a'>AC08088803</subfield></datafield><datafield tag='035' ind1=' ' ind2=' '><subfield code='a'>(AT-OBV)AC08088803</subfield><subfield code='a'>(Aleph)007994064ACC01</subfield><subfield code='a'>(DE-599)OBVAC08088803</subfield></datafield><datafield tag='245' ind1='0' ind2='0'><subfield code='a'>&lt;&lt;Die&gt;&gt; Internationale Werkbundsiedlung Wien 1932</subfield><subfield code='c'>hrsg. von Josef Frank</subfield></datafield></record>"
```

Comment:
1.) the xml should be moved to a separated file
2.) the path to the file should then be given over the cli e.g. @click.option("--xml"). where the file is the default file, but the user could set also other files.